docs/GettingStartedDocs/Contributors/LinuxInstallInfo.md

```diff
@@ -53,7 +53,7 @@ If you also specified a `CMAKE_INSTALL_PREFIX`, this would install the SDK to
 /foo/opt/openenclave
 ```
 
-## Create a redistributable SDK package
+## Create a redistributable SDK Debian package
 
 To create a redistributable package (e.g. deb, rpm), use `cpack`. Specify
```

Comment:
we should probably remove the rpm from here if that doesn't currently work

Reply:
+1, either remove Debian from the title, or change the line to 
"To create a redistributable Debian package, use `cpack`."

Reply:
Fixed.